include/event_manager/event_manager.h

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/** @file
+ * @brief Event manager header.
+ */
+
+#ifndef ZEPHYR_INCLUDE_EVENT_MANAGER_EVENT_MANAGER_H_
+#define ZEPHYR_INCLUDE_EVENT_MANAGER_EVENT_MANAGER_H_
+
+
+/**
+ * @defgroup event_manager Event Manager
+ * @brief Event Manager
+ *
+ * @{
+ */
+
+#include <zephyr.h>
+#include <zephyr/types.h>
+#include <sys/reboot.h>
+#include <sys/__assert.h>
+#include <logging/log_ctrl.h>
+
+#include <event_manager/event_manager_priv.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/** @def SUBS_PRIO_MIN
+ *
+ * @brief Index of the highest subscriber priority level.
+ */
+#define SUBS_PRIO_MIN    _SUBS_PRIO_FIRST
```

Comment:
since it is a public api it should be prefixed with EVENT_MANAGER, or at least obvious abbreviation (`EVT_MGR_`?)

Reply:
if we use `app_event` approach that would become `APP_EVENT_SUBS_PRIO_MIN`